src/main/webapp/sass/_core.scss

```diff
@@ -3443,4 +3444,14 @@ div.border {
         width: 600px;
         flex-grow: 1;
     }
+}
+
+.MissingAnnotations {
+    color: $synapse-gray-med;
+}
+.ValidAnnotations {
+ color: $synapse-green;
+}
+.InvalidAnnotations {
+  color: $synapse-yellow;
 }
```

Comment:
Could/should these be reused? Maybe .ValidIcon, .InvalidIcon, etc.

Reply:
You could use existing generalized classes.  .synapse-green, .synapse-yellow.  Probably need to make one for .synapse-gray-med